src/GraphQL/GraphQL.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net6.0</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>netstandard2.1;netstandard2.0;net6</TargetFrameworks>
```

to align with other usages